e2e/tests/localAuthorityMaintainsCaseAfterSubmission_test.js

```diff
@@ -51,41 +50,3 @@ Scenario('local authority uploads court bundle', async (I, uploadDocumentsEventP
   caseViewPage.selectTab(caseViewPage.tabs.documents);
   I.seeDocument('Court bundle', 'mockFile.txt');
 });
-
-Scenario('local authority provides a statements of service', async (I, caseViewPage, loginPage, addStatementOfServiceEventPage) => {
```

Comment:
We are no longer doing statement of service in submission state?

Reply:
See previous